src/profile/templates/user_management/profile/delete-profile.html

```diff
@@ -0,0 +1,9 @@
+<div class="notification-header">
+    <span ng-show="!failed" translate>You are about to permanently delete your profile and all associated data. This cannot be undone. Please confirm you would like to progress.</span>
```

Comment:
Can this be proceed rather than progress: `You are about to permanently delete your profile and all associated data. This cannot be undone. Please confirm you would like to proceed.` (you will need to update the `pot` and `po` files - sorry I didn't spot that before)